src/client/components/Paginator/Paginator.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import ReactPaginate from 'react-paginate'
+
+type Props = {
+  className: string
+  onPageChange?(selectedItem: { selected: number }): void
+  pageCount: number
+  pageRangeDisplayed: number
+}
+
+const Paginator: React.FC<Props> = ({ className, onPageChange, pageCount, pageRangeDisplayed }) => (
+  <ReactPaginate
+    className={className}
+    breakLabel="..."
+    nextLabel=">"
+    onPageChange={onPageChange}
+    pageRangeDisplayed={pageRangeDisplayed}
+    pageCount={pageCount}
+    previousLabel="<"
+    renderOnZeroPageCount={null}
+  />
+)
+
+Paginator.defaultProps = {
+  onPageChange: () => null,
```

Comment:
this shouldnt be optional i think
i cant think of any case where we would use the component without this